pyproject.toml

```
@@ -0,0 +1,126 @@
+[build-system]
+requires = ["setuptools>=64", "wheel", "setuptools_scm[toml]>=3.4.3"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "oculy"
+dynamic=["version"]
+description = "Visualisation app for Enaml based plugin applications"
+readme = "README.rst"
+requires-python = ">=3.7"
+license = {file = "LICENSE"}
+authors = [
+  {name = "Matthieu C. Dartiailh", email = "m.dartiailh@gmail.com"}
+]
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "License :: OSI Approved :: BSD License",
+    "Natural Language :: English",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+]
+
+[options]
+packages = "find:"
+zip_safe = false
+long_description = "file: README.rst"
+long_description_content_type = "text/x-rst"
+platforms = "Linux; Windows; Mac"
+
+dependencies = [
+    "glaze",
```

Since there is no way to specify a default optional dependencies and people are likely to forget to pick a qt backend let's stick to Qt6 with the pyside bindings.
```suggestion
    "glaze[qt6-pyside]",
```

---

👍